docs/appendix/machine-specific-install.rst

```diff
@@ -1,8 +1,8 @@
 Machine-specific Install
 ========================
 
-NCAR HPC Cheyenne
------------------
+NCAR HPC Cheyenne or Casper
```

Comment:
Changing this section title broke a reference to it [elsewhere](https://github.com/NOAA-CSL/MELODIES-MONET/blame/b9aee5c0fd3f9d2212306e663169c9db40e27655/docs/tutorial/installation.rst#L29), that's why the check failed. Casper is mentioned below, so maybe it would be ok to leave the heading?

Reply:
This title is mentioned in docs/tutoriall/installation.rst. We can add Casper we should just update it in both places.